dependencies.py

```diff
@@ -68,11 +68,11 @@ def extra_requirements() -> typing.Dict[str, typing.List[str]]:
             "google-cloud-storage~=1.20",
             # because of storey which isn't compatible with google-cloud-bigquery >3.2, conflicting grpcio
             # google-cloud-bigquery 3.3.0 has grpcio >= 1.47.0, < 2.0dev while storey 1.2.2 has grpcio<1.42 and >1.34.0
-            "google-cloud-bigquery[pandas]>=3.2,<3.5",
+            "google-cloud-bigquery[pandas, bqstorage]~=3.2",
```

Comment:
I don't understand the comment above this line. It looks to me like it's irrelevant anymore since we allow google-cloud-bigquery 3.3.0